### ISSUE TITLE
Add images to any message (Chat)?

### ISSUE THREAD

**Issue by aolko** [LostRuins/koboldcpp#732]
Can i add multiple images and can i add images to any message (including my own, not just replies)?

**Comment by LostRuins** [comment]
Yes you can. Click the "Add Img" button and you can add an image at any time, and you can reposition images in edit mode.

**Comment by aolko** [comment]
Can i trigger that via text?

**Comment by LostRuins** [comment]
You can set images to "Autogenerate" and it will generate a new image every few hundred words.  
What do you mean by "trigger via text"?

**Comment by aolko** [comment]
> You can set images to "Autogenerate" and it will generate a new image every few hundred words. What do you mean by "trigger via text"?

just like you'd do in gpt-4 - "draw me a...", "generate a...", "give me a photo of a..."

**Comment by aolko** [comment]
Oh and can it expand WI entries when sending the prompt to a111?
